third-party/llvm/Makefile.include-llvm

```diff
@@ -17,5 +17,5 @@ LLVM_CLANG_LIBS=-lclangFrontend -lclangSerialization -lclangDriver -lclangCodeGe
 LLVM_CXXFLAGS=$(shell $(LLVM_INSTALL_DIR)/bin/llvm-config --cxxflags | sed 's/ -O[0-4s] / /') -DHAVE_LLVM
 LLVM_CFLAGS=$(shell $(LLVM_INSTALL_DIR)/bin/llvm-config --cflags | sed 's/ -O[0-4s] / /') -DHAVE_LLVM
 LLVM_INCLUDES=-I$(LLVM_INCLUDE_DIR) -I$(LLVM_CLANG_CODEGEN_INCLUDE_DIR)
-LLVM_LIBS=-L$(LLVM_LIB_DIR) $(LLVM_CLANG_LIBS) $(LLVM_LLVM_LIBS)
+LLVM_LIBS=-L$(LLVM_LIB_DIR) $(LLVM_CLANG_LIBS) $(LLVM_LLVM_LIBS) -lLLVMSupport -ldl
```

Comment:
These changes are in the right file - but I suspect that -lLLVMSupport -ldl is only needed for 3.3. Why did you add these?